pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -3265,6 +3265,19 @@ type CephBlockPoolRadosNamespaceList struct {
 	Items           []CephBlockPoolRadosNamespace `json:"items"`
 }
 
+// CephBlockPoolRadosNamespaceSpecMirroring represents the mirroring configuration of CephBlockPoolRadosNamespaceSpec
+type CephBlockPoolRadosNamespaceSpecMirroring struct {
+	// Enabled is a flag to enable or disable mirroring
+	// +optional
+	Enabled bool `json:"enabled"`
+	// RemotePool is the name of the secondary cluster pool
+	// +optional
+	RemoteNamespace string `json:"remoteNamespace"`
```

Comment:
Isnt this need to be `[]string` as i can setup `1:N` mirroring in the blockPool?

Reply:
@Madhu-1 From what I recall this is not supported by ceph. @nbalacha Please correct me if I am wrong.

Reply:
> @Madhu-1 From what I recall this is not supported by ceph.

1:N mirroring is supported, but the remote namespace for a particular local namespace is configured and stored globally, not for each peer individually.  We don't have existing CLI or API support for configuring something on a per-peer basis.  This isn't new, the namespace remapping feature just brings it into the spotlight.

For now, this can be a simple string -- an array of strings wouldn't be quite right anyway, if we had the support in RBD it would probably need to be a map from peer UUIDs to strings.

Reply:
Thanks, @idryomov for confirmation.